README.rst

```diff
@@ -117,6 +117,17 @@ These are both equivalent to running::
 
     codespell --quiet-level 3 --count --skip "*.po,*.ts,./src/3rdParty,./src/Test"
 
+If several config files are present, they are read in the following order:
+
+#. ``pyproject.toml``
+#. ``setup.cfg``
+#. ``.codespellrc``
+#. any additional file supplied via ``--config``
+
+If a codespell configuration is supplied in several of these files,
+the configuration from the most recently read file overwrites previously
+specified configurations.
```

Comment:
This is true in practice, but nowhere documented in theory:
https://github.com/python/cpython/issues/96765

Reply:
it is documented in https://docs.python.org/3/library/configparser.html

> It is possible to read several configurations into a single [ConfigParser](https://docs.python.org/3/library/configparser.html#configparser.ConfigParser), where the most recently added configuration has the highest priority. Any conflicting keys are taken from the more recent configuration while the previously existing keys are retained.

Reply:
I'm convinced by https://github.com/python/cpython/issues/96765#issuecomment-1288117030. :+1: